frontend/Makefile

```
@@ -0,0 +1,7 @@
+# pnpm --config.confirmModulesPurge=false prevents pnpm from blocking on
+# stdin in the continuous build.
+
+all:
+	npm install -g pnpm
+	pnpm install --config.confirmModulesPurge=false
+	pnpm build
```

```suggestion
	pnpm build
	pnpm build-storybook
```

I'd suggest building the storybook too so we can serve it alongside the app as well